src/components/EditorSideBar/EditorSideBarContent.tsx

```diff
@@ -3,16 +3,18 @@ import React from "react";
 import { RiArrowLeftSLine } from "react-icons/ri";
 import Blueprints from "./Blueprints";
 import Folder from "./Folder";
+import { useGetCourtsQuery } from "@/redux/api/courtSizeApi";
+import { ICourtData } from "@/interfaces/design";
 import { IconContext } from "react-icons";
 
 interface Props {
   iconClickTitle: string;
   onHandleCloseClick: () => void;
 }
-const showContainerItem = (iconClickTitle: string) => {
+const showContainerItem = (iconClickTitle: string, data: ICourtData[]) => {
```

Comment:
The parameter name "data" can change to some specific name like courtData or anything you like. We use "data" only when we use it once and drop it later, but this component use data multiple times.